app/src/main/res/layout/tab2_fragment.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
if this layout is exactly the same as tab2_fragment, just delete it and use tab1_fragment as the layout for both fragments